app/controllers/account_details.js

```diff
@@ -0,0 +1,111 @@
+import Ember from "ember";
+import AjaxPromise from 'browse/utils/ajax-promise';
+const { getOwner } = Ember;
+
+export default Ember.Controller.extend({
+  queryParams: ['orgId'],
+
+  authenticate:Ember.inject.controller(),
+  messageBox: Ember.inject.service(),
+  i18n: Ember.inject.service(),
+  organisationId: Ember.computed.alias('model.organisation.id'),
+  organisationsUserId: Ember.computed.alias('model.organisationsUser.id'),
+  position: "",
+
+  userTitle: Ember.computed('model', function() {
+    let userTitle = this.get('model.user.title');
+    let titles = this.get('titles');
+
+    if(userTitle) {
+      let filteredUserTitle = titles.filter((item) => userTitle === item.id);
+      return { name: filteredUserTitle[0].name.string, id: userTitle };
+    } else {
+      return { name: titles.get('firstObject.name').string, id: 'Mr' };
```

Comment:
I think this is the default condition, is no title exist then we show `Mr`, right?
If that so than I think we can keep it outside else block as default. 
AS else block statement won't get executed because if condition has return statement